pytest.ini

```diff
@@ -1,4 +1,4 @@
 [pytest]
-addopts = -s -vv -x --reruns 5 --reruns-delay 10 --durations=100
+addopts = -s -vv -x --reruns 5 --reruns-delay 10 --durations=20
```

Comment:
the number of returned tests i believe are a bit too many. maybe fetching only the top 20 is more useful.